templates/includes/header.html

```diff
@@ -10,27 +10,12 @@
         {% if link.section %}
         {% set href = "/" ~ link.section ~ "/" %}
         {% set section = get_section(path=link.section ~ "/_index.md") %}
-        <details {{ util::current_open(if_path_starts_with=href) }} class="section-wrap mobile">
-            <summary class="section {{ util::current_class(if_path_starts_with=href) }}">
-                {{ link.title }}
+        <div class="section-wrap">
+            <input id="{{ link.slug }}-submenu-checkbox" type="checkbox" class="submenu-checkbox" aria-hidden="true">
+            <div class="submenu-title-container">
+                <label for="{{ link.slug }}-submenu-checkbox" class="submenu-title">{{ link.title }}</label>
                 <div class="arrow"></div>
```

Comment:
Why is the arrow not inside the `<label>`?

Reply:
Mostly to use `submenu-title-container` as a `flex` so I had more room to play with the arrow itself. Having a `div` inside a `label` also felt a little wrong.

Should I remove the container div and move the arrow inside the label?

Reply:
I think you should yes, as right now tapping the arrow doesn't expand the menu.